### PR TITLE
New version: Microsoft.PowerBI version 2.124.1805.0

### PR DESCRIPTION
**Pull request by PckgrBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135582)

### CODE REVIEW

File: manifests/m/Microsoft/PowerBI/2.124.1805.0/Microsoft.PowerBI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: -silent -norestart
+  SilentWithProgress: -passive -norestart
+  InstallLocation: INSTALLLOCATION="<INSTALLPATH>"
+  Log: -log "<LOGPATH>"
+  Upgrade: -update
+  Custom: ACCEPT_EULA=1
```
Comment (Trenly) [inline manifests/m/Microsoft/PowerBI/2.124.1805.0/Microsoft.PowerBI.installer.yaml:22]:
```suggestion
```
@denelon for policy and working with PowerBI team

**Comment by Trenly** [comment]
[Policy] Blocking-Issue

**Comment by Exorcism0666** [comment]
- Resolve: #135731

**Comment by denelon** [comment]
We're testing the agreements validation logic, and we anticipate pushing this into production in early February.

**Pull request closed without merging**